### ISSUE TITLE
Vernemq pod keep CrashLoopBackOff in Kubernetes when using helm

### ISSUE THREAD

**Issue by lmq1999** [vernemq/docker-vernemq#214]
I follow the site here: https://medium.com/@mauri660/kubernetes-mqtt-broker-autoscale-with-custom-metric-1cb9aee0686e

And the github to install vernemq on k8s

But it keep CrashLoopBackOff, I did accept EULA

Here is the log:

```
08:46:40.853 [info] Application sext exited with reason: stopped
08:46:40.853 [info] Application riak_dt exited with reason: stopped
08:46:40.853 [info] Application eleveldb exited with reason: stopped
08:46:40.854 [error] Loading of /vernemq/bin/../lib/vmq_plumtree-1.10.2/ebin/vmq_plumtree.beam failed: not_purged
08:46:40.859 [warning] The on_load function for module eleveldb returned:
{error,{load_failed,"Failed to load NIF library: 'Error loading shared library libsnappy.so.1: No such file or directory (needed by /vernemq/bin/../lib/eleveldb-2.0.2+build.552.ref06fefb8/priv/eleveldb.so)'"}}

```

I see the problems in #199 fix the libsnappy but i still got error, any help ?

**Comment by codeadict** [comment]
I think the issue here is when helm pulls it is getting an older version `1.10.2-alpine` which had the libsnappy issue, the corrected version was `1.10.2-1-alpine` and we need to push that one as the default. For now as a workaround you can set `image.tag` to `1.10.2-1-alpine`. Thanks!

**Comment by codeadict** [comment]
@lmq1999 the default image now should have this fixed.
